enhancements/sig-architecture/68-token-registration/README.md

```diff
@@ -0,0 +1,157 @@
+## Release Signoff Checklist
```

Comment:
Please add https://github.com/open-cluster-management-io/api/blob/main/docs/clusterjoinprocess.md as a see-also.  It may be worth it to move the doc into this repo.

That doc outlines the current process and a few key features of the flow are visible there:
1. the subject has a group associated with the managedcluster and the name has an agent associated with the process on that managedcluster
2. an agent on the managedcluster is able to get new agent credentials by using shared (non-agent) credentials to create a request and the hub admin can approve or reject.
3. an agent on the managedcluster is able renew credentials to identify itself.  This also allows for non-shared credential flows.
4. different agents on the managed cluster are members of the same group and have different names, with the agent clearly identified
5. the hub admin gets a choice about whether or not to allow a particular managecluster to get a valid credential
6. the hub admin can control expiry/removal of a particular agent.  In this case, by rotating the signing credential.  (Different distros are better or worse at doing this.)

Any new flow that is developed should allow the same capabilities and should use the standard group and usernames to ease adoption.  Capability 2 may be negotiable (not required), but it would be limiting because the hub would need to communicate unique credentials to each managed cluster.

Reply:
👍 Added to implementation details section